website/content/api-docs/system/lease-count-quotas.mdx

```diff
@@ -26,12 +27,19 @@ that can either be a namespace or mount.
 - `path` `(string: "")` - Path of the mount or namespace to apply the quota.
   A blank path configures a global lease count quota. For example `namespace1/`
   adds a quota to a full namespace, `namespace1/auth/userpass` adds a quota to
-  `userpass` in `namespace1`. Updating this field on an existing quota can have
-  "moving" effects. For example, updating `auth/userpass` to
-  `namespace1/auth/userpass` moves this quota from being a global mount quota to a
-  namespace specific mount quota. Quotas on a non-root namespace are not inherited by child
+  `userpass` in `namespace1`, and `namespace1/kv-v2/data/foo/bar` adds a quota to
+  a specific secret on a K/V v2 mount in `namespace1`. A trailing glob (`*`) can also
+  be added as part of the path after the mount to match paths that share the same prefix
+  prior to the glob. `namespace1/kv-v2/data/foo/*` would match both `namespace1/kv-v2/data/foo/bar`
+  and `namespace1/kv-v2/data/foo/baz`. Updating this field on an existing
+  quota can have "moving" effects. For example, updating `namespace1` to
+  `namespace1/auth/userpass` moves this quota from being a namespace quota to a
+  namespace specific mount quota. Non-global quotas are not inherited by child
   namespaces.
 - `max_leases` `(int: 0)` - Maximum number of leases allowed by the quota rule.
+- `role` `(string: "")` - If set on a quota where `path` is set to an auth mount with a
+  concept of roles (such as `/auth/approle/`), this will make the quota restrict login
+  requests to that mount that are made with the specified role.
```

Comment:
This explanation implies that the role will effectively be ignored if the auth mount does not have a concept of roles. Would it make sense to explicitly state that here?

Reply:
The quota creation would actually fail, but you're right, I think we should state that explicitly - thanks for the suggestion!

Reply:
Ah, gotcha. That makes sense. Thanks :).